zsh-aws-vault.plugin.zsh

```diff
@@ -13,6 +13,35 @@ function avsh() {
   aws-vault exec $1 -- zsh
 }
 
+function aws-login() {
+    aws-vault login $1
+}
+
+function aws-login-incognito() {
+    aws-get-login-link $1 | xargs /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --args --incognito --new-window
+}
+
+function aws-get-login-link() {
+    aws-vault login -s $1
+}
+
+function aws-profiles() {
```

Comment:
This is interesting - since it's not a direct aws-vault method, I wonder if we could do something like `av-profiles`? Feedback welcome.